tests/utils/test_env_pack.py

```diff
@@ -67,12 +67,24 @@ def test_pack_env_for_databricks_model_serving_pip_requirements(tmp_path, mock_d
         )
     )
 
+    # Create a mock environment directory
+    mock_env_dir = tmp_path / "mock_env"
+    mock_env_dir.mkdir()
+    (mock_env_dir / "lib").mkdir()
+    (mock_env_dir / "lib" / "python3.8").mkdir()
+    (mock_env_dir / "lib" / "python3.8" / "site-packages").mkdir()
+    (mock_env_dir / "lib" / "python3.8" / "site-packages" / "test_package").mkdir()
+    (
+        mock_env_dir / "lib" / "python3.8" / "site-packages" / "test_package" / "__init__.py"
+    ).write_text("")
```

Comment:
can we create a dummy environment using `venv` and use it?

Reply:
@serena-ruan 

```
venv.create(mock_env_dir)
```

is all we need to run.

Reply:
ohhh ok! I thought you're referring to enabling the virtual env 😅 let me update